node_modules/oae-activity/emailTemplates/mail.html.jst

```diff
@@ -33,15 +33,37 @@
      * Render the email footer
      */
     var renderFooter = function() {
-        var copyrightSymbolI18n = util.i18n.translate('__MSG__COPYRIGHT_SYMBOL__');
-        var apereoFoundationI18n = util.i18n.translate('__MSG__APEREO_FOUNDATION__');
+        var oaeNameI18n = util.i18n.translate('__MSG__OPEN_ACADEMIC_ENVIRONMENT__');
         var url = baseUrl + '/me?emailpreferences';
 
+        var oaeLink = '<a href="http://www.oaeproject.org">' + oaeNameI18n + '</a>';
+        var instanceLink = (instance.URL) ? '<a href="' + instance.URL + '">' + util.html.encodeForHTML(instance.name) + '</a>' :
```

Comment:
The suggested code can't be done because the `instance.name` has to be run through an `encodeForHTML` function. I feel using a tertiary statement is the most easy to read version of this logic.